couchbase/src/main/scala/akka/stream/alpakka/couchbase/scaladsl/OperationResult.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.couchbase.scaladsl
+
+import akka.stream.alpakka.couchbase.impl.FailedOperation
+
+import scala.util.Try
+
+/** Scala API */
+final case class SingleOperationResult[T](entity: T, result: Try[String])
```

Comment:
Do we need different classes for the Java and the Scala API?
Wouldn't wrapping getter-methods be enough for a nice Java API?

As in
https://github.com/akka/alpakka/blob/7404ed625228f46a48b8be1e7f4ed512662c91c4/reference/src/main/scala/akka/stream/alpakka/reference/model.scala#L23-L60